packages/eth-rpc-cache/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eth-rpc-cache",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
If someone is using v1.0.0 with a custom strategy, these changes will most probably break their code. We should bump it to 2.0.0 to be safe.

Reply:
version updated [fc17775](https://github.com/bloq/eth-packages/pull/6/commits/fc1777506dcbdb09a6c043fbbe886b5946f267f0)